### PR TITLE
[hack] Add relative PR merge times to errata script

### PR DESCRIPTION
**Pull request by mtnbikenc**
Adds a new column (Merged) which represents the relative time since the PR was merged.  Also, the Commit Hash is reduced to an abbreviated commit to save space with the additional column.

```
Merge commits in release-4.15 after build windows-machine-config-operator-container-10.15.0-41
Commit   Merged         PR#  PR Title                                                          Jira ID       Fix Version   JIRA Status   Errata Status
======== ============== ==== ================================================================= ============= ============= ============= =============
9581032b 4 hours ago    2024 [release-4.15] OCPBUGS-27503: Use DSR load balancing in kube-prox OCPBUGS-27503 4.15.0        MODIFIED      Not Attached
971f3ede 6 hours ago    2030 [release-4.15] [test] Open SMB port when creating share                                                                  

Merge commits in release-4.15 since WMCO Branching Day 10.15.0 included in build windows-machine-config-operator-container-10.15.0-41
Commit   Merged         PR#  PR Title                                                          Jira ID       Fix Version   JIRA Status   Errata Status
======== ============== ==== ================================================================= ============= ============= ============= =============
5147e86f 8 days ago     2015 [release-4.15] OCPBUGS-27239: [cluster] Keep comma as delimiter i OCPBUGS-27239 4.15.0        Verified      Attached
9c39c7bb 8 days ago     2008 [release-4.15] submodule update 01/12                                                                                    
1c93443e 12 days ago    2003 [release-4.15] [controllers] Introduce err check when merging CA                                                         
20830119 12 days ago    1996 [release-4.15] Add support for Nutanix in machineset.sh                                                                  
2fbfb383 13 days ago    1984 [release-4.15] OCPBUGS-25755,OCPBUGS-25756: Fix network config sc OCPBUGS-25755 4.15.0        Verified      Attached
46363314 2 weeks ago    1976 [release-4.15] WINC-1181: Add pod CPU and Memory metrics          WINC-1181     WMCO 10.15.0  Closed        Attached
48dbeaac 5 weeks ago    1977 [release-4.15] OCPBUGS-25350: Ensure WICD service account token   OCPBUGS-25350 4.15.0        Verified      Attached

```

### CODE REVIEW

**Comment by sebsoto** [comment]
/approve

This will be a helpful change
I tested it and it works as advertised.

File: hack/errata-check.sh [hunk 1]
```diff
@@ -229,7 +229,8 @@ function list-commits {
     local jira_status=""
     local attached=""
     commit_hash=$(echo "$commit" | awk '{ print $1 }')
-    pr_num=$(echo "$commit" | awk '{ print substr($5,2) }')
+    commit_date=$(echo "$commit" | awk -F" " '{ print $2,$3,$4 }')
```
Comment (sebsoto) [inline hack/errata-check.sh:232]: This could be made a `local` variable as all the others are

Comment (mtnbikenc) [reply hack/errata-check.sh:233]: Updated.

**Comment by sebsoto** [comment]
/override ci/prow/aws-e2e-operator ci/prow/aws-e2e-upgrade ci/prow/azure-e2e-operator ci/prow/azure-e2e-upgrade ci/prow/gcp-e2e-operator ci/prow/nutanix-e2e-operator ci/prow/platform-none-vsphere-e2e-operator ci/prow/vsphere-e2e-operator ci/prow/vsphere-proxy-e2e-operator ci/prow/wicd-unit-vsphere

**Pull request merged**